### ISSUE TITLE
update for stretch?

### ISSUE THREAD

**Issue by 2600box** [fpunktk/raspi-fde#2]
thanks for putting this together. i am trying to do the same with raspbian stretch and couldn't make it work.
i noticed some things had changed, like the disk references are now the uuid, etc. and there are probably others that I missed.

it would be great if you could update your work for stretch. i am sure many would make use of it.

cheers

**Comment by fpunktk** [comment]
I agree, an update would be cool. I actually put it together because I could not find a recent tutorial, too :-)
The problem is: I don't have time at the moment and I don't use the setup anymore. But perhaps this request encourages me to do it ;-)
